initializr-service/src/main/resources/application.yml

```diff
@@ -163,8 +163,10 @@ initializr:
         mappings:
           - versionRange: "[1.3.0.RELEASE, 1.5.0.M1)"
             version: 7.7.7
-          - versionRange: "1.5.0.M1"
-            version: 8.4.1
+          - versionRange: "[1.5.0.M1, 2.0.0.M1)"
+            version: 8.4.4
+          - versionRange: "2.0.3.RELEASE"
```

Comment:
I believe that this should be 2.0.0.RELEASE so that there isn't a hole in the ranges.

Reply:
I guess I could change it so, but it wont have a functional difference as only the latest stable (currently 2.0.3) is offered anyways? I'll do it anyways.

Reply:
Or actually it could be M1...

Reply:
Yeah, I think that's fine too. As long as there are no gaps that a `.RELEASE` could fall though, I believe Stéphane will be happy.